isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp

```diff
@@ -468,19 +468,29 @@ namespace Isis {
   /**
    * This method write out the labels and image data to the specified output file.
    * Creates an IMG and XML file.
-   *
-   * @param outFile QString of the name of the output file. Will create an XML 
+   * 
+   * @param outFile QString of the name of the output image file. Will create an XML 
    *        and an IMG file with the output file name.
    *
    */
   void ProcessExportPds4::WritePds4(QString outFile) {
     
     FileName outputFile(outFile);
 
-    QString path(outputFile.originalPath());
+    // Name for output label
+`   QString path(outputFile.originalPath());
```

Comment:
It looks like there's an extra back tick here

Reply:
Thanks for catching that. I'm not sure how it slipped in in-between testing and committing.

Reply:
@jessemapel Okay, fixed!